### PR TITLE
Minor grammar fixes to the developer guide

### PR DESCRIPTION
**Pull request by NoahvdAa**

### CODE REVIEW

File: docs/paper/dev/event-api/custom-events.md [hunk 1]
```diff
@@ -5,11 +5,11 @@ slug: /dev/custom-events
 # Custom Events
 
 Creating custom events is a great way to add functionality to your plugin. 
-This will allow for other people to listen for your custom events and add functionality to your plugin.
+This will allow other people to listen to your custom events and add functionality to your plugin.
```
Comment (Owen1212055) [inline docs/paper/dev/event-api/custom-events.md:8]: I'd say "other plugins to listen", not players.

Comment (NoahvdAa) [reply docs/paper/dev/event-api/custom-events.md:8]: It says other people, but the point is still valid, thanks :)

**Review by olijeffers0n — APPROVED** [review]
Your telling me my gramer wa'int perfekt frist time around. outraigous

File: docs/paper/dev/event-api/custom-events.md [hunk 2]
```diff
@@ -5,11 +5,11 @@ slug: /dev/custom-events
 # Custom Events
 
 Creating custom events is a great way to add functionality to your plugin. 
-This will allow for other people to listen for your custom events and add functionality to your plugin.
+This will allow other plugins to listen to your custom events and add functionality to your plugin.
 
 ## Creating a custom event
 
-To create a custom event, you need to create a class that extends `Event`. Each event has a `HandlerList` that contains all the listeners that are listening for that event.
+To create a custom event, you need to create a class that extends `Event`. Each event should have a `HandlerList` that will contain all the listeners that are listening to that event.
```
Comment (kennytv) [inline docs/paper/dev/event-api/custom-events.md:12]: requires, not should have

**Review by kennytv — APPROVED** [review]

**Pull request merged**